httprequester.go

```diff
@@ -1,13 +1,6 @@
 package yoti
 
-import (
-	"bytes"
-	"fmt"
-	"io/ioutil"
-	"log"
-	"net/http"
-)
-
+// Deprecated
```

Comment:
Please could you add why this has been deprecated - something like `HttpMethod is now passed to SignedRequest as a string` . Optionally you could add when it will be removed as well, something like `will be removed in v3.0.0`